src/net/http.rs

```diff
@@ -80,6 +95,21 @@ lazy_static! {
     static ref PATH_GETMICROBLOCKS_CONFIRMED : Regex = Regex::new(r#"^/v2/microblocks/confirmed/([0-9a-f]{64})$"#).unwrap();
     static ref PATH_GETMICROBLOCKS_UNCONFIRMED : Regex = Regex::new(r#"^/v2/microblocks/unconfirmed/([0-9a-f]{64})/([0-9]{1,5})$"#).unwrap();
     static ref PATH_POSTTRANSACTION : Regex = Regex::new(r#"^/v2/transactions$"#).unwrap();
+    static ref PATH_GET_ACCOUNT: Regex = Regex::new(&format!(
+        "^/v2/accounts/(?P<principal>{})$", *PRINCIPAL_DATA_REGEX)).unwrap();
+    static ref PATH_GET_MAP_ENTRY: Regex = Regex::new(&format!(
+        "^/v2/map_entry/(?P<address>{})/(?P<contract>{})/(?P<map>{})$",
+        *STANDARD_PRINCIPAL_REGEX, *CONTRACT_NAME_REGEX, *CLARITY_NAME_REGEX)).unwrap();
+    static ref PATH_POST_CALL_READ_ONLY: Regex = Regex::new(&format!(
+        "^/v2/contracts/call-read/(?P<address>{})/(?P<contract>{})/(?P<function>{})$",
+        *STANDARD_PRINCIPAL_REGEX, *CONTRACT_NAME_REGEX, *CLARITY_NAME_REGEX)).unwrap();
+    static ref PATH_GET_CONTRACT_SRC: Regex = Regex::new(&format!(
+        "^/v2/contracts/source/(?P<address>{})/(?P<contract>{})$",
+        *STANDARD_PRINCIPAL_REGEX, *CONTRACT_NAME_REGEX)).unwrap();
+    static ref PATH_GET_CONTRACT_ABI: Regex = Regex::new(&format!(
+        "^/v2/contracts/interface/(?P<address>{})/(?P<contract>{})$",
+        *STANDARD_PRINCIPAL_REGEX, *CONTRACT_NAME_REGEX)).unwrap();
+    static ref PATH_GET_TRANSFER_COST: Regex = Regex::new("^/v2/fees/transfer$").unwrap();
```

Comment:
Should we have a `v2/ping` / `v2/health_check`?

Reply:
Yes, eventually, but probably not in this PR.